### ISSUE TITLE
A question about how to compute the Retrieval metrics (R@1, R@5,...)

### ISSUE THREAD

**Issue by liqing-ustc** [cesc-park/CRCN#7]
As stated in the paper, "_Each algorithm retrieves the best sequences from **training database** for a query image sequence, ...  Since the training and test data are disjoint, each algorithm can only retrieve similar (but not identical) sentences at best_", then how should I compute the Retrieval metrics (R@1, R@5,..)?
@cesc-park can you help me with the question? Thanks a lot.

**Comment by cesc-park** [comment]
CRCN model basically computes a score between an image stream and a sentence sequence. 
So, you can compute the score for a given test image stream between sentence sequences(one by one) in our test set including the sentence sequence correspond to the given test image stream. After that, sort them by their scores. Then, you can get ranks.

**Comment by liqing-ustc** [comment]
@cesc-park Thanks for your quick reply. Based on what you said, the performance of the retrieval metrics will be largely influenced by **the size of the test set**? I mean that if the test set is small, it is likely to have a higher number of the retrieval metrics, am I right?

**Comment by cesc-park** [comment]
Rank scores can be influenced by the size of the test set, But we used normalized scores.

**Comment by liqing-ustc** [comment]
@cesc-park How can I normalize the rank scores? Specifically, how should I normalize Recall@1?

**Comment by cesc-park** [comment]
Sorry, you're right. We can't normalize R@1 by itself. I want to say that we report the mean R@1.
If the test set is much smaller than other test sets, it is likely to have a higher number of the retrieval metrics.

**Comment by liqing-ustc** [comment]
I see. Thanks a lot!

**Comment by liqing-ustc** [comment]
@cesc-park BTW, do you publish the datasets Disney and NYC?
